lib/querystring.js

```diff
@@ -5,6 +5,18 @@
 const QueryString = exports;
 const Buffer = require('buffer').Buffer;
 
+// This constructor is used to store parsed query string values. Instantiating
+// this is faster than explicitly calling `Object.create(null)` to get a
+// "clean" empty object
+function ParsedQueryString() {}
+ParsedQueryString.prototype = Object.create(null, {
+  constructor: {
```

Comment:
`constructor` is removed now.

IMHO I think more people are going to be using `typeof foo === 'object'` rather than `foo instanceof Object`. Perhaps @ChALkeR can give us some insight about this?